astroquery/casda/core.py

```diff
@@ -274,7 +324,7 @@ def stage_data(self, table, verbose=False):
 
         return self._complete_job(job_url, verbose)
 
-    def cutout(self, table, *, coordinates=None, radius=None, height=None,
+    def cutout(self, table, *, coordinates=None, radius=1*u.arcmin, height=None,
```

Comment:
I think this was accidentally pulled in with one of  the merge commits, please rebase to get rid of those and any unrelated changes.

Reply:
The commit was intentional - just cleaning up a minor aspect that was missed in the last PR